docs/polkadotjs_apps_local.md

```diff
@@ -1,26 +1,35 @@
 ---
 id: polkadotjs_apps_local 
-title: Connect to local node 
+title: Connect to a local node 
 ---
 
 import useBaseUrl from '@docusaurus/useBaseUrl';
 
-*Prerequisites: your node is running on your local machine or you have port `9944` forwarded to your local machine.*
+You can use the Polkadot.JS app to connect to your local HydraDX node. For this purpose, you need to have access to port `9944` which is used for RPC websocket connections.
```

Comment:
The naming is inconsistent. It's called Polkadot/apps everywhere in the docs. If we want to use this kind of naming, we should change it everywhere else, but then `polkadot.js.org/apps` is a more correct way to name it.